projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-flat/json-editor-node-flat/json-editor-node-flat.component.html

```diff
@@ -44,11 +44,11 @@
         <ng-container *ngIf="schema?.type === 'string'">
           <!-- No format -->
           <div *ngIf="!schema.format">
-            <ngx-input *ngIf="!schema?.enum" type="text" [ngModel]="model" (change)="updateModel($event)"
+            <ngx-input *ngIf="!schema?.enum" type="text" [placeholder]="placeholder" [ngModel]="model" (change)="updateModel($event)"
               [requiredIndicator]="false" [required]="required">
             </ngx-input>
 
-            <ngx-select [filterable]="false" *ngIf="schema?.enum" [ngModel]="[model]"
+            <ngx-select [filterable]="false" [placeholder]="placeholder" *ngIf="schema?.enum" [ngModel]="[model]"
```

Comment:
Hmm, not sure if it makes sense here since the user is already selecting from a predefined list of values.

Reply:
can the select ever be in a state where nothing is selected? Thats when the placeholder would come in handy so it should matter if they are selecting from a list just if the value can be undefined.

Reply:
oh I see we haven't implemented placeholders within ngx-select

Reply:
I was going to say to leave the placeholder. But it’s not supported then?